src/sgame/sg_bot_parse.cpp

```diff
@@ -136,6 +136,18 @@ static AIValue_t currentWeapon( gentity_t *self, const AIValue_t* )
 	return AIBoxInt( BG_GetPlayerWeapon( &self->client->ps ) );
 }
 
+// Return true if currently selected weapon is ready
+// This won't work for all cases since some alien "modes"
+// do not necessarily block all other modes.
+// It might also be a problem for mass driver's scope, but
```

Comment:
Mass driver scope is not a weapon. It's a client-side shader thing changing the view.

Reply:
does not means that the zoom in there is handled like a weapon. BUT I agree with your comment.

Reply:
I know for a fact that zoom is implemented in the cgame. So please remove the inaccurate comment.